lib/core/client.js

```diff
@@ -626,15 +626,7 @@ function onSocketError (err) {
     while (client.pending && client[kQueue][client[kPendingIdx]].host === client[kHost]) {
       client[kQueue][client[kPendingIdx]++].onError(err)
     }
-  } else if (
-    !client.running &&
-    err.code !== 'ECONNRESET' &&
-    err.code !== 'ECONNREFUSED' &&
-    err.code !== 'EHOSTUNREACH' &&
-    err.code !== 'EHOSTDOWN' &&
-    err.code !== 'UND_ERR_SOCKET' &&
-    err.code !== 'UND_ERR_INFO'
```

Comment:
UND_ERR_INFO should probably still be here

Reply:
Should the client try to reconnect with those error codes?